storage/SystemInfo/Arch.h

```diff
@@ -52,6 +52,7 @@ namespace storage
 
 	bool is_ppc_mac() const { return ppc_mac; }
 	bool is_ppc_pegasos() const { return ppc_pegasos; }
+        bool is_ppc_power_nv() const { return ppc_power_nv; }
```

Comment:
Since they normally capitalize the "NV" as in "PowerNV", it's really two words, isn't it?